lms/djangoapps/edraak_university/helpers.py

```diff
@@ -1,11 +1,11 @@
 from django.contrib.auth.models import AnonymousUser
-
+from django.utils.translation import get_language
 from opaque_keys.edx.locator import CourseLocator
 
-from models import UniversityID
 
 
 def get_university_id(user, course_id):
+    from models import UniversityID
```

Comment:
This was because of overlapping imports. Once the translation function was moved out of the helper this line was put back to its place. Done.